### ISSUE TITLE
Finish UDF support for Contacts - SetContactUDF

### ISSUE THREAD

**Issue by matt40k** [SIMSBulkImport/API#12]
Current version reads as:

```csharp
       public bool SetContactUDF(Int32 personid, string udfvalue)
       {
           // TODO 
           return false;
        }
```

**Comment by matt40k** [comment]
Issue also raised on EduGeek by howartp@ - http://www.edugeek.net/forums/sims-bulk-import/154731-udfs-v2-3-19-a.html.

**Comment by matt40k** [comment]
Resolved with 52617a616cb29e5d6aa8ead4a51e07a226d47d13
